web/src/views/home/sections/motivation_section.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Container } from 'reactstrap';
+
+const MotivationSection = () => (
+  <section className="section mt-4">
+    <Container>
+      <h2 className="mt-3">Why Would I Analyze My Sleep?</h2>
+      <p className="lead text-justify">
+        Although we spend a major part of our day sleeping, we cannot truly understand what happens during this time, as
+        sleep is inherently an altered state of consciousness. With Polydodo, we to offer an accessible way to answer,
+        for each moment of the night, how was my sleep.
+      </p>
+      <p className="text-justify">
+        We do so by offering you a picture of your sleep based on the five standard sleep stages defined by the American
+        Academy of Sleep Medicine (AASM) Manual for the Scoring of Sleep and Associated Events. These sleep stages are
+        Wake, Rapid-Eye Movement (REM), and the three non-REM stages, which goes from the lightest, N1, to the deepest,
+        N3. Polydodo scores every 30 seconds epochs of your night into these five stages.
+      </p>
+      <p className="text-justify">
+        By studying your own sleep, you may discover how your daily habits affect your sleep architecture. For each
+        night that you will use Polydodo, it will generate a sleep report that includes metrics such as:
```

Comment:
```suggestion
        By studying your own sleep, you may discover how your daily habits affect your sleep architecture. During each
        night you use Polydodo, it will generate a sleep report that includes metrics such as:
```

Reply:
Il me semble que c'est pas durant chaque nuit, plus qu'à la fin de chaque nuit qu'on obtient le sleep report.

Reply:
At the end of each night?